api-reference/10 UI Components/dxChart/1 Configuration/argumentAxis/breaks/breaks.md

```diff
@@ -14,7 +14,7 @@ A scale break allows breaking off a part of the axis to improve the readability
 
 ![DevExtreme HTML5 JavaScript Charts Scale Breaks](/images/ChartJS/visual_elements/scale-breaks_arg-axis.png)
 
-Each object in the **breaks** array configures a single scale break. Note that a scale break is visible only if its range exceeds the tick interval.
+Each object in the **breaks** array configures a single scale break. We recommend that a scale break range exceeds the tick interval at least twice. Otherwise, the scale break might not be visible.
```

Comment:
`We recommend that a scale break range exceeds the tick interval at least twice.`

Can we update this to the following?

`A scale break range should exceed the tick interval at least twice.`

Reply:
If a tickInterval is 100, the range between a scale break's startValue and endValue should be at least 200. Like this:
breaks: [{startValue: 300, endValue: 500}]

Reply:
Thanks for the clarification. I think I understand now. What do you think about the following suggestion?

`A scale break range should span at least two tick intervals.`

Reply:
I'm not sure about the word 'span'. It implies something visual but we are talking about actions to be done in code. So we need to compare two numeric values - a scale break's range and a tick interval.